README.md

```diff
@@ -1,84 +1,55 @@
-# razorpay-php
+<p align="center"><img src="https://razorpay.com/images/logo.png"></p>
```

Comment:
I never found someone who reads the readme in the editor after installing the package/framework, etc. Even if that's the case this readme will be in the razorpay/razorpay-php directory so that's obvious its your readme.